test/es-module/test-esm-wrappers-live-binding.mjs

```diff
@@ -0,0 +1,16 @@
+// Flags: --experimental-modules
+
+import '../common';
+import assert from 'assert';
+
+import fs, { readFile } from 'fs';
+
+const fn = () => {};
+
+assert.strictEqual(fs.readFile, readFile);
+
+fs.readFile = fn;
```

Comment:
Can we also include a test of this mutation from the CommonJS side?

Reply:
Can there be a test for 
```js
import fs from "fs"
import require from "./require.js"

assert.strictEqual(require("fs"), fs)
```

Reply:
thats already tested https://github.com/nodejs/node/pull/18131/files#diff-6ab2e2802cef62e5073aeefa6037159fR17

Reply:
They should still be `===` without unwrapping the proxy. Userland won't be using that helper.

Reply:
userland doesn't need to check the equality?

Reply:
>  and actually proxies don't have equality with the object they wrap.

Ah bummer. I thought I checked that out. Oh well if that bit can't be `===` then it's okay.

Reply:
Since `===` isn't the result for either a proxy or an object of getter/setters I guess either could go. I do dig the simplicy of a proxy though.